### PR TITLE
Add GPU transform and DiceCE training pipeline

### PR DESCRIPTION
**Pull request by Maeve816**
The changes made:

To use DiceCE, in src/py/nets/segmentation.py : 
  weight=torch.tensor(self.hparams.ce_weight) to ce_weight=torch.tensor(self.hparams.ce_weight)
  (somehow the whole function get updated)

To adjust to the pkl file format, in src/py/loaders/tt_dataset.py:
  add function TTDataModuleSegPklTrans (resize/rotate/zoom included, on CPU, suffers from computational inefficiency
  add function TTDataModuleSegPklGPUTrans (enables GPU-based transformations to improve training efficiency)
  add function TTDataModuleSegPklGPUResize (refine the GPU-based pipeline by enforcing correct tensor shapes, ensuring compatibility with Dice/DiceCE losses and stable training behavior)

To create my own virtual env, changed tt_environment.yml

Please let me know if there are any comments.

### CODE REVIEW

**Comment by juanprietob** [comment]
1. New training for segmentation experiment with labels for the different regions
